### PR TITLE
Added community partner Filip Rakowski

### PR DESCRIPTION
**Pull request by filrak**
Added myself as a community partner
image

### CODE REVIEW

**Comment by phanan** [comment]
@filrak Thanks for the PR! Can you resolve the merge conflicts, though?

**Comment by filrak** [comment]
@phanan should be fine now ;)

**Comment by phanan** [comment]
👍

File: src/v2/guide/team.md [hunk 1]
```diff
@@ -1136,7 +1136,41 @@ order: 803
       links: [
         'https://vuejs.london'
       ]
-    }
+    },
+    {
+      name: 'Phan An',
+      title: 'Backend Designer & Process Poet',
```
Comment (oohira) [inline src/v2/guide/team.md]: @filrak @phanan Is it a mistake that @phanan has been added here?

Comment (phanan) [reply src/v2/guide/team.md]: Oh, indeed.

**Pull request merged**